rosetta/retriever/retriever.go

```diff
@@ -136,23 +141,31 @@ func (r *Retriever) Balances(rosBlockID identifier.Block, rosAccountID identifie
 	// Get the Cadence value that is the result of the script execution.
 	amounts := make([]object.Amount, 0, len(symbols))
 	for _, symbol := range symbols {
+
+		// We get the script to get the vault balance and execute it.
```

Comment:
```suggestion
		// We generate the script to get the vault balance and execute it.
```

Nitpick but sounds better IMO, avoids the repetition of `get` and is more accurate